docker/Dockerfile.debian

```
@@ -1,11 +1,23 @@
-FROM golang AS builder
-
-RUN mkdir /src
-ADD . /src
-
-RUN cd /src && \
-	mkdir -p output/bin && \
-	make V=1 bin/step
+FROM --platform=$BUILDPLATFORM golang:alpine AS builder
```

You should build using [debian.](golang:bullseye)
```suggestion
FROM --platform=$BUILDPLATFORM golang:bullseye AS builder
```